### DISCUSSION TITLE
[YOUR-PASSWORD] means what ?

### DISCUSSION THREAD

**Discussion by rajanshresth** [supabase/supabase#13199] [Questions] [answered]
postgresql://postgres:[YOUR-PASSWORD]@???.supabase.co:5432/postgres

**Comment by GaryAustin1** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/54564956/227744436-59a93428-39f7-463c-bfb5-7617b25fa3fe.png)
It is the password you entered when you set up your instance, or you can reset in settings...
